params/network_upgrades.go

```diff
@@ -4,14 +4,18 @@
 package params
 
 import (
+	"time"
+
 	"github.com/ava-labs/avalanchego/utils/constants"
 	"github.com/ava-labs/subnet-evm/utils"
 )
 
 var (
+	defaultActivationTime = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
+
 	LocalNetworkUpgrades = MandatoryNetworkUpgrades{
 		SubnetEVMTimestamp: utils.NewUint64(0),
-		DUpgradeTimestamp:  utils.NewUint64(0),
+		DUpgradeTimestamp:  utils.TimeToNewUint64(defaultActivationTime),
```

Comment:
can we change from handling it this way and updating the vm clock to only overriding mandatory network upgrades for specific networkIDs?

I think we could make this change a separate PR as well, since some tests are currently enabling the DUpgrade initially even when using genesisJSONSubnetEVM

Reply:
reverted for now.